classes/stageAssignment/StageAssignmentDAO.inc.php

```diff
@@ -176,16 +210,17 @@ function insertObject($stageAssignment) {
 		$this->update(
 			sprintf(
 				'INSERT INTO stage_assignments
-					(submission_id, user_group_id, user_id, date_assigned, recommend_only)
+					(submission_id, user_group_id, user_id, date_assigned, recommend_only, can_change_metadata)
 				VALUES
-					(?, ?, ?, %s, ?)',
+					(?, ?, ?, %s, ?, ?)',
 				$this->datetimeToDB(Core::getCurrentDate())
 			),
 			array(
 				$stageAssignment->getSubmissionId(),
 				$this->nullOrInt($stageAssignment->getUserGroupId()),
 				$this->nullOrInt($stageAssignment->getUserId()),
-				$stageAssignment->getRecommendOnly()?$stageAssignment->getRecommendOnly():0
+				$stageAssignment->getRecommendOnly()?$stageAssignment->getRecommendOnly():0,
+				$stageAssignment->getCanChangeMetadata()?$stageAssignment->getCanChangeMetadata():0
```

Comment:
These should both be `(int)` casts -- not sure why the convoluted syntax :)

Reply:
Probably have pasted the syntax from somewhere else.

Changed!